src/qt/governancelist.cpp

```diff
@@ -135,8 +136,10 @@ QVariant ProposalModel::data(const QModelIndex& index, int role) const
             return proposal->startDate().date();
         case Column::END_DATE:
             return proposal->endDate().date();
-        case Column::PAYMENT_AMOUNT:
-            return proposal->paymentAmount();
+        case Column::PAYMENT_AMOUNT: {
+            return BitcoinUnits::floorWithUnit(m_display_unit, proposal->paymentAmount() * COIN, false,
```

Comment:
hm, I just noticed that `proposal->paymentAmount()` returns float here.

`str_amounts` is assigned as string `ValueFromAmount(payment.nAmount))` which doesn't lose precision, but conversion back through float and multiplying to COIN can potentionallly add a strange 0.00000002 at the end, I think.

Is it possible to avoid conversion to `float` here?
```
        if (UniValue amountValue = find_value(prop_data, "payment_amount"); amountValue.isNum()) {
            m_paymentAmount = amountValue.get_real(); // <----- here
        }
```

Reply:
Good catch! We should be using `double` here. Will fix in a follow-up PR.